docs/getting_started/installation_guide/platforms/karamel.rst

```diff
@@ -0,0 +1,87 @@
+.. _karamel-installer:
+
+=======================
+Karamel
+=======================
+
+#. Download and install `Karamel <http://www.karamel.io/>`_ 
+#. Run Karamel.
+#. Click on the "Load Cluster Definition" menu item in Karamel. You are now prompted to select a cluster definition YAML file.
+
+Example cluster definitions can be found on `github <https://github.com/logicalclocks/karamel-chef/tree/master/cluster-defns>`_.
+
+
+**Choosing which services to run on which nodes**
+
+You need to decide which services you will install on which nodes. In Karamel, we design a set of *Node Groups*, where each *Node Group* defines a stack of services to be installed on a machine. Each machine will only have one *Node Group* set of services.
+Here are some recommended setups:
+
+* a *single node cluster* that includes all services on a single node.
+* a *tiny cluster* set of *heavy* stacks that includes a lot of services on each node.
+* a *small cluster* set of *heavy* stacks that includes lots of services on each node.
+* a *large cluster* set of *light* stacks that includes fewer services on each node.
+
+**Single Node Setup**
+You can run the entire Hopsworks application platform on a single node. You will have a NodeGroup with all of your services on the single node.
+
+**Tiny Cluster Setup**
+
+We recommend the following setup that includes the following NodeGroups, and requires at least 2 nodes to be deployed:
+
+#. Hopsworks, MySQL Server, NDB Mgmt Server, HopsFS NameNode, YARN ResourceManager, NDB Data Node, Kafka, hopslog (ELK stack), hopsmonitor (prometheus)
+#. HopsFS DataNode, YARN NodeManager
+
+This is really only a test setup, but you will have one node dedicated to YARN applications and file storage, while the other node handles the metadata layer services.
+
+**Small Cluster Setup**
+
+We recommend the following setup that includes three NodeGroups:
+
+#. Hopsworks, MySQL Server, NDB Mgmt Server, Kafka, hopslog (ELK stack), hopsmonitor (prometheus)
+#. HopsFS NameNode, YARN ResourceManager, MySQL Server, NDB Data Node
+#. HopsFS DataNode, YARN NodeManager
+
+.. _ElasticHA: https://www.elastic.co/guide/en/elasticsearch/reference/master/high-availability.html
+   
+A highly available small cluster would require at least two instances in the last three NodeGroups. Note that  Elasticsearch needs to be extra configuration if it is to be sharded across many instances, see Elasticsearch documentation for details (ElasticHA_).
```

Comment:
Should be rewritten to "A highly available small cluster would require at least two instances in every NodeGroup".

Also there is an extra space between that and Elasticsearch.

Reply:
fixed